### PR TITLE
Enable packages delete

### PR DESCRIPTION
**Pull request by troydai**
@pranavkm 

I'll merge tonight.

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -52,8 +52,7 @@ use-standard-lifecycle
           {
               Log.Info("Removing " + package);
 
-              // Check in the try run first to ensure correct packages are removed.
-              // Directory.Delete(package); 
+              Directory.Delete(package, true);
```
Comment (Eilon) [inline makefile.shade]: Named param please!

Comment (troydai) [inline makefile.shade]: Yes, sir.

**Comment by troydai** [comment]
Per discussion with @pranavkm this is not a perfect solution. Deleting packages and republish open up a 2 minutes window in which other repo build may fail. I'll retract this one. I'll work out another strategy.

**Comment by pranavkm** [comment]
We could delete things that are not part of the current Coherence build - would be a safer change.

**Pull request closed without merging**